config/activator-ds.yaml

```diff
@@ -0,0 +1 @@
+./core/activator-ds.yaml
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
./core/activator-ds.yaml

```

Reply:
I think links don't work that way :/ ?